charts/kured/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.6.1"
 description: A Helm chart for kured
 name: kured
-version: 2.4.2
+version: 2.4.3
```

Comment:
The convention is to make chart changes *after* this functional PR has merged, so take a stash of these changes (all changes under `charts/`) and have them ready to submit as a discrete PR after the changes to `cmd/kured/main.go` merge.

Reply:
Done.